### PR TITLE
Brazil 2.0

### PR DESCRIPTION
**Pull request by francinum**
## About The Pull Request

"fixes" the brazil algorithm to prevent systems going offscreen.

## Why It's Good For The Game

Probably isn't.

## Changelog
:cl:
tweak: brazil
/:cl:

### CODE REVIEW

**Comment by Bokkiewokkie** [comment]
🇧🇷 a z il

**Comment by MajorMeltdown** [comment]
i want to get off of mr bones wild sector 3

**Comment by Kmc2000** [comment]
Neat! This is a much nicer way of doing it, mine was pretty scuffed

File: nsv13/code/controllers/subsystem/starsystem.dm [hunk 1]
```diff
@@ -786,19 +786,29 @@ To make things worse, this hellhole is entirely RNG, so good luck mapping it!
...
+		randy.x = rand(1, 200)
+		randy.y = rand(1, 100)+30 //Offset vertically for viewing 'pleasure'
+		while(randy.x in used_x)
+			randy.x = rand(1, 100)
+			conflictcount++
+		while(randy.x in used_y)
+			randy.y = rand(1, 100)+30
+			conflictcount++
+		used_x.Insert(randy.x)
+		used_y.Insert(randy.y)
```
Comment (DeltaFire15) [inline nsv13/code/controllers/subsystem/starsystem.dm:811]: This doesn't work due to how Insert() functions, and if that did work it'd create an infinite loop instead.

**Comment by theboy6545** [comment]
TA: Can we stop using the brazil meme?
Fran and KMC: Brazil go BRRRR

**Review by Kmc2000 — APPROVED** [review]

**Review by Kmc2000 — APPROVED** [review]

**Pull request merged**